src/controllers/expenseController.js

```diff
@@ -0,0 +1,102 @@
+const {
+  filterExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+  getExpenses,
+} = require('../services/expenses.service');
+const { getUser } = require('../services/users.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
```

Comment:
Consider validating the query parameters to ensure they are in the expected format before passing them to `filterExpenses`. This will help prevent potential errors or unexpected behavior.